crates/bevy_render/src/lib.rs

```diff
@@ -469,9 +471,10 @@ unsafe fn initialize_render_app(app: &mut App) {
                     render_system,
                 )
                     .in_set(RenderSet::Render),
-                World::clear_entities.in_set(RenderSet::Cleanup),
+                World::clear_entities.in_set(RenderSet::FinalCleanup),
             ),
-        );
```

Comment:
Can `RenderSet::FinalCleanup` not be configured by adding it to the `.chain()` on line 158?

Reply:
Ugh, this shouldn't be part of this PR. Good call though!